webapp/src/components/data_selectors/jira_commentvisibility_selector/jira_commentvisibility_selector.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {ReactSelectOption} from 'types/model';
+
+import BackendSelector, {Props as BackendSelectorProps} from '../backend_selector';
+
+const stripHTML = (text: string) => {
+    if (!text) {
+        return text;
+    }
+
+    const doc = new DOMParser().parseFromString(text, 'text/html');
+    return doc.body.textContent || '';
+};
+
+type Props = BackendSelectorProps & {
```

Comment:
If we add a new component to any existing project we make sure it is a functional component as class components are old and not supported

Reply:
Made a ticket for this task [MI-2384](https://brightscout.atlassian.net/browse/MI-2384), and will continue when this pr is approved.